src/widgets/lv_img.c

```diff
@@ -610,6 +618,14 @@ static void draw_img(lv_event_t * e)
         bg_pivot.y = img->pivot.y + ptop;
         lv_area_t bg_coords;
 
+
+        int32_t tiled = lv_obj_get_style_bg_img_tiled(obj, LV_PART_MAIN);
```

Comment:
`bg_img` is a different thing. We should have "image fit" properties dedicated to the images but's it's an other story. 
So let's make all vector image object sized.

Reply:
Ok.